modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/QueryProcessor.java

```diff
@@ -35,7 +55,8 @@ public interface QueryProcessor extends IgniteComponent {
      * @return List of sql cursors.
      *
      * @throws IgniteException in case of an error.
-     * */
+     */
```

Comment:
Let's rewrite tests to remove deprecated methods

Reply:
Deprecated methods can't be removed right now because the JDBC event handler still relies on them.  I would suggest to return to this when we will be implementing a script support

just removed the deprecation for now